requirements.adoc

```diff
@@ -188,6 +240,16 @@ the PACS to the imaging modality. The imaging modality will put the Study
 Instance UID in the images it creates. This allows the {project-short-name} to
 access the DICOM study and its images in the PACS via the Study Instance UID.
 
+```
+### Judy : Remember we have a Study , then series and then instance... so a CT Head is a study , then it can be axial , coronal or sagittal (the series) and each image in the series is an instance
+- Summarizing the above i think of the following functionality
+1. Ability to schedule an exam (this is in the orderable data where the date of examination is selected)
+2. Ability to create a modality worklist that can be queried by day
+3. Ability to cancel the examination -- if the patient cannot pay or dies -- and it gets the status cancelled
+4. Generate an accession ID (or a study ID that we can track for reporting)
+5. Support performance of a study by the technologist and set the status to completed both in RIS and using the MPPS .
+```
```

Comment:
totally agree with 3-5, these features are reflected in the user stories, if not will be (MPPS feature for example will be done later).